be/src/Project/Services/Client/Client.Api/DependencyInjection/CompositionRoot.cs

```diff
@@ -11,6 +12,7 @@ public static class CompositionRoot
         public static void RegisterClientApiDependencies(this IServiceCollection services)
         {
             services.AddScoped<IClientsService, ClientsService>();
+            services.AddScoped<IGroupsService, GroupsService>();
```

Comment:
**suggestion:** Rename to services.AddScoped<IClientGroupsService, ClientGroupsService>();